packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -750,7 +750,7 @@ describe('TokenDetectionController', () => {
   describe('AccountsController:selectedAccountChange', () => {
     let clock: sinon.SinonFakeTimers;
     beforeEach(() => {
-      clock = sinon.useFakeTimers();
+      clock = useFakeTimers();
```

Comment:
Why do we need to make this change ?

Reply:
Noticed a small lint warning : 'useFakeTimers' not found in imported namespace 'sinon'.eslint[import-x/namespace](https://github.com/un-ts/eslint-plugin-import-x/blob/v4.6.1/docs/rules/namespace.md)
when using` clock = sinon.useFakeTimers();`
So i updated that to import useFakeTimers from sinon directly